syntax = "proto3";

/* Status codes */
enum StatusCode {
  STATUS_OK = 0;
  STATUS_UNAUTHORIZED = 1;
}